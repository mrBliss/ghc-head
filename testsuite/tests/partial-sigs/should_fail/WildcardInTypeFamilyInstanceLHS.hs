{-# LANGUAGE PartialTypeSignatures, TypeFamilies, InstanceSigs #-}
module WildcardInTypeFamilyInstanceLHS where

class Foo k where
  type Dual k :: *

instance Foo Int where
  type Dual _ = Maybe Int
