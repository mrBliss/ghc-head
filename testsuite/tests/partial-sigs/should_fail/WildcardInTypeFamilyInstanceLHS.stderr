
WildcardInTypeFamilyInstanceLHS.hs:8:13:
    Wildcard not allowed
    In type family instance equation of ‘Dual’: Dual _
